tajo-cli/src/main/java/org/apache/tajo/cli/tsql/DefaultTajoCliOutputFormatter.java

```diff
@@ -206,4 +278,33 @@ public static String parseErrorMessage(String message) {
 
     return message;
   }
+
+  /**
+   * borrowed from Presto
+   */
+  private static boolean detectRealTerminal()
```

Comment:
Oh, I see. When I borrowed this method, I wondered about fitness of this comments to tajo. Currently, This comments are written for Presto, not Tajo. So I just confused about this comments to include this PR. But, After Testing this PR in many individual environment, I take conviction about this method, but I missed to include its comments. This is my mistake. I'll add comments.